pkg/dao/repository_config_test.go

```diff
@@ -967,13 +967,13 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 			URL:  pointy.String(""),
 		}, { // Providing a valid url & name
 			Name: pointy.String("Some Other Name"),
-			URL:  pointy.String("http://foobar.com"),
+			URL:  pointy.String("http://foobar.com/"),
```

Comment:
The gives ERROR 403: Forbidden. I could not find a special use domain name that provides that. Do you need that error here?

Reply:
In this case we're not actually using the URL for its response, we're mocking the response. What we use it for is to parse a string with URL-style syntax. So it's okay that the URL doesn't work.